sdk/spring/azure-spring-cloud-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfigurationTest.java

```diff
@@ -26,8 +27,13 @@
     private static final String HOST = "localhost";
     private static final int PORT = 6379;
     private static final boolean IS_SSL = true;
-    private ApplicationContextRunner contextRunner =
-        new ApplicationContextRunner().withConfiguration(AutoConfigurations.of(AzureRedisAutoConfiguration.class));
+    private ApplicationContextRunner contextRunner;
```

Comment:
1. Why change this in this PR?
2. How about change contextRunner to local variable in each test mechod?

Reply:
> Why change this in this PR?

In pipline, this test failed occasionally.

>How about change contextRunner to local variable in each test method?

I think it's good.